lib/src/github/models.rs

```diff
@@ -96,6 +96,40 @@ pub struct App {
     pub name: String,
 }
 
+#[derive(Deserialize, Serialize, Clone, Debug)]
+pub struct Team {
+    pub name: String,
+    pub slug: String,
+    pub html_url: String,
+}
+
+impl Team {
+    pub fn new(org: &str, slug: &str) -> Team {
+        Team {
+            name: slug.to_string(),
+            slug: slug.to_string(),
```

Comment:
if these are the same value, maybe we should just include it once, either as `name` or as `slug` (though, "name" makes more sense to me :)

Reply:
Yeah, we probably don't have a need for the `name` field. I probably want to keep it `slug` since that's what the GH API calls it.